x-pack/plugins/fleet/common/services/validate_package_policy.ts

```diff
@@ -393,6 +393,8 @@ export const countValidationErrors = (
 ): number => {
   const flattenedValidation = getFlattenedObject(validationResults);
   const errors = Object.values(flattenedValidation).filter((value) => Boolean(value)) || [];
+  // eslint-disable-next-line no-console
```

Comment:
I will assume that you are not merging this main and are deploying this to a QA environment to test.

Reply:
No We're not merging this.